src/validation/signup.js

```diff
@@ -0,0 +1,66 @@
+import { check, validationResult } from 'express-validator';
+import { signupErrors } from '../utils/constants/errorMessages';
+import pullErrors from '../utils/helpers/pullErrors';
+import models from '../models';
+
+/**
+ * The signup controller
+ * @param { object } req - The request object
+ * @param { object } res - The response object
+ * @param { function } next - Pass to next middleware
+ * @return { void }
+ */
+const signupValidator = [
+  check('firstname')
+    .exists({ checkFalsy: true })
+    .withMessage(signupErrors.undefinedFirstName)
+    .isAlpha()
+    .withMessage(signupErrors.invalidFirstName),
+  check('lastname')
+    .exists({ checkFalsy: true })
+    .withMessage(signupErrors.undefinedLastName)
+    .isAlpha()
+    .withMessage(signupErrors.invalidLastName),
+  check('email')
+    .exists({ checkFalsy: true })
+    .withMessage(signupErrors.undefinedEmail)
+    .isEmail()
+    .withMessage(signupErrors.invalidEmail)
+    .matches(/andela.com$/)
```

Comment:
I believe this won't catch an email like this `james@bondandela.com`. You can update the regex to `/@andela.com$/` instead

Reply:
Thank you for the insight.